src/actions/alerts.js

```diff
@@ -0,0 +1,16 @@
+import * as ActionTypes from '../actiontypes'
+
+const AlertActions = {
+  showError(error, message = null) {
+    console.error(error)
+    return dispatch => {
+      dispatch({ type: ActionTypes.SHOW_ERROR, message: (message || error.message) })
```

Comment:
Like `ValidatorsActions.add`, this one should be synchronous and just return the action object. The same for `reset`.

Reply:
done